lib/3scale/backend/logging/external/impl/bugsnag.rb

```diff
@@ -56,7 +54,7 @@ def configure(api_key)
                   config.timeout = 3
                   config.logger = Backend.logger
                   config.meta_data_filters = []
-                  config.notify_release_stages = ['production', 'preview']
+                  config.notify_release_stages = ['production', 'staging', 'development']
```

Comment:
This is deprecated, the new alternative is `enabled_release_stages`
https://rubydoc.info/gems/bugsnag/Bugsnag/Configuration#notify_release_stages-instance_method

Reply:
Fixed!

Reply:
OK, but while you were fixing this, I left another comment :sweat_smile:  https://github.com/3scale/apisonator/pull/373#discussion_r1514314333

WDYT?